Xamarin.Forms.Platform.Android/Renderers/ListViewRenderer.cs

```diff
@@ -284,13 +284,18 @@ void OnScrollToRequested(object sender, ScrollToRequestedEventArgs e)
 			if (Element.IsGroupingEnabled)
 			{
 				var results = templatedItems.GetGroupAndIndexOfItem(scrollArgs.Group, scrollArgs.Item);
```

Comment:
Small thing but when we get the results from the GetGroupAndIndexOfItem can we assign Item1 and Item2 to local variables that have a meaning eg indexOfGroup and indexOfItem

Reply:
@nickrandolph : Thanks for this suggestion. I added meaningful local variables.